nhs/data/handling.py

```diff
@@ -29,13 +30,13 @@ def read_csv(file_path: str) -> pl.LazyFrame | None:
 
 @logger.catch()
 @log_entry_exit()
-def read_xlsx(file_path: str) -> pl.LazyFrame | None:
+def read_xlsx(file_path: str) -> dict[str, DataFrame]:
     """
     Load a .xlsx file into a polars `LazyFrame`, returning None if exception occurs
 
     **NOTE**: Polars use `xlsx2csv` to read .xlsx files, so whole CSV file is read
     """
-    return pl.read_excel(file_path).lazy()
+    return pl.read_excel(file_path, sheet_id=0)
```

Comment:
Don't hard-code `sheet_id=0`, pass it as a parameter `sheet_id: None | int = None`  instead and do the same for `read_csv` and `read_psv`. Add it as an argument to `read_spreadsheet` which will pass it to whatever spreadsheet reader we get

Reply:
And change the return type to `dict[str, LazyFrame] | LazyFrame`, and call `.lazy()` 

(making it lazy is more for consistency with the other functions, XLSX by default is a `DataFrame` but we'll need to add that type to `read_spreadsheet` which will be tedious when we use the function, it'll be easier to make it lazy so we don't need to change any of that)